notebooks/.env

```diff
@@ -1,5 +1,4 @@
-# This is a sample .env file with variables used in neomaril codex
-NEOMARIL_URL='https://neomaril.staging.datarisk.net'
-NEOMARIL_USER='email@email.com'
-NEOMARIL_PASSWORD='pass'
-NEOMARIL_GROUP_TOKEN='TOKEN'
\ No newline at end of file
+MLOPS_URL='http://localhost:7070'
+MLOPS_USER='johnrc692@gmail.com'
+MLOPS_PASSWORD='teste@123'
+MLOPS_GROUP_TOKEN='TODO'
```

Comment:
do these credentials work?

Reply:
They work locally, but it was not supose to go there

Reply:
https://github.com/datarisk-io/mlops_codex/pull/64/commits/4eb4cd115b8408e34c538664bf9b9f25a99eb714